libs/types/src/tokens.rs

```diff
@@ -183,7 +183,18 @@ where
 /// enables us to use the `TrancheCurrency` type separately where solely this
 /// enum variant would be relevant. Most notably, in the `struct Tranche`.
 #[derive(
-	Clone, Copy, PartialOrd, Ord, PartialEq, Eq, Debug, Encode, Decode, TypeInfo, MaxEncodedLen,
+	Clone,
+	Copy,
+	Default,
```

Comment:
I do not like default feature on this. Can we hide it behind `std` or `runtime-benchmarks`?

Reply:
Can we 👍🏻

Reply:
Actually, after using the last cool bench utility `bench_default_investment_id()` (thanks @wischli for abstracting this), the default requirement in benchmarking is no longer needed. 

Thanks for pointing this out!